### PR TITLE
Added CSS to Leadership Team section

### PR DESCRIPTION
**Pull request by Yeen125**
Added the CSS via SASS to the Leadership Team section of the About page. Also made minor changes to the HTML code to better fit the Figma mock-up.

### CODE REVIEW

**Review by daleinen7 — CHANGES_REQUESTED** [review]
Good start. Just a few notes and then the design is a little off. 
pr
design
Looks like when it's two rows, the bottom two are spread further apart than in the design. 
Screen Shot 2021-05-15 at 3 54 37 PM
And then the heading part doesn't look like it's the same size as the "Work with Us" section that it's based off of.

File: scss/main.scss [hunk 1]
```diff
@@ -1549,4 +1549,24 @@ p.cardText {
...
+#our-team {
+  background-color: #f5f5f6;
+  padding: 30px;
+
+  header {
+    text-align: center;
+  }
+
+  h2, h3, p {
+    color: #333;
```
Comment (jrapala) [inline scss/main.scss:1564]: Please use a variable

File: pages/about.html [hunk 2]
```diff
@@ -5,4 +5,44 @@
...
+            <img alt="picture of Joseph Tajaran" src="\images\proxy-img.png">
+            <!--Replace proxy image when actual picture becomes avaiable-->
+            <h3>Joseph Tajaran</h3>
+            <p>Brigade Co-Captain</p>
+        </div>
+        <div class="leader-card">
+            <img alt="picture of Dan Morr" src="\images\proxy-img.png">
+            <!--Replace proxy image when actual picture becomes avaiable-->
+            <h3>Dann Morr</h3>
+            <p>Paternships Co-Lead</p>
```
Comment (jrapala) [inline pages/about.html:33]: Typo

**Review by jrapala — CHANGES_REQUESTED** [review]
Is your other PR, https://github.com/Code-For-Chicago/code-for-chicago-jekyll/pull/139, no longer relevant? This seems to use a lot of the same code you have there.

If so, please close the other PR.

**Pull request closed without merging**